server.js

```diff
@@ -0,0 +1,7 @@
+var express = require('express')
+var app = express()
+app.get('/ping', (req, res) =>{
+    res.send('Hello World')
+})
+
+app.listen(3000);
```

Comment:
Always end your files with a newline to avoid any potential issues with programs that expect or require them. [medium]